src/test/java/uk/gov/digital/ho/hocs/info/api/StandardLineServiceTest.java

```diff
@@ -91,5 +95,7 @@ public void shouldCreateStandardLineExpiringPrevious(){
         verify(documentClient).deleteDocument(standardLine.getDocumentUUID());
         verifyNoMoreInteractions(standardLineRepository);
         verifyNoMoreInteractions(documentClient);
+        verify(caseworkClient, times(1)).clearCachedStandardLineForTopic(uuid);
```

Comment:
time(1) is not needed here, by default it is going to assumed is should only call it once.

Reply:
thanks